extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_linkAuthentication.php

```diff
@@ -1,6 +1,6 @@
 <?php if ( isset( $model['subtitle'] ) ): ?>
 	<div class="wds-global-navigation__account-menu-dropdown-caption"><?= DesignSystemHelper::renderText( $model['subtitle'] ) ?></div>
 <?php endif; ?>
-<button class="wds-button wds-is-full-width <?= isset( $model['subtitle'] ) ? 'wds-is-secondary' : ''; ?>">
+<a href="<?= Sanitizer::encodeAttribute( $href ); ?>" id="<?= Sanitizer::encodeAttribute( $model['title']['key'] ); ?>" class="wds-button wds-is-full-width <?= $model['title']['key'] === 'global-navigation-anon-register' ? 'wds-is-secondary' : ''; ?>">
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)